src/invoice2data/extract/templates/be/be.accor.invest.ibis.yml

```diff
@@ -0,0 +1,73 @@
+issuer: Accor Invest Belgium nv SA
+fields:
+  amount:
+    parser: regex
+    regex: Total to pay EUR\s+(\d?.?\d{1,4}[,]\d{2})
+    type: float
+  amount_tax:
+    parser: regex
+    regex: \d{2}[%]\s+\w\s+\d{1,4}[,]\d{2}\s+(\d{1,4}[,]\d{2})
```

Comment:
You can probably just use `%` instead of `[%]` (same applies to `[,]` and other regexes below)